### DISCUSSION TITLE
Explorer Styling Issue

### DISCUSSION THREAD

**Discussion by Posvar** [microsoft/SandDance#669] [Q&A] [answered]
I'm trying to deploy the explorer on my own domain - testing first on desktop environment. I was able to load the explorer on localhost:3000 but the styling was all messed up (e.g. all the divs (topbar, scrollable container, sanddance-view, etc.) just stacked vertically on top of each other. I assumed it was missing CSS so I downloaded and added these to my <head>
    <link rel="stylesheet" type="text/css" href="/sanddance-app.css">
    <link rel="stylesheet" type="text/css" href="/sanddance-app-site.css?2">

However - this is how it's now formatting (entire experience is confined to top 60 or so pixels):
<img width="530" alt="image" src="https://github.com/microsoft/SandDance/assets/37360346/2778bcb9-fb28-4b16-aefb-e504a0b3028a">
Best way to recreate what I'm seeing is to load the [Demo Vote](https://microsoft.github.io/SandDance/app/) sample and then delete this element from HTML head: 
<img width="338" alt="image" src="https://github.com/microsoft/SandDance/assets/37360346/f0f235b3-01c7-4514-8142-e67082e03d16">
Any help appreciated.

**Comment by danmarshall** [comment, marked as answer]
You might take a look at these related issues: 
* https://github.com/microsoft/SandDance/issues/305#issuecomment-774955652
* https://github.com/microsoft/SandDance/issues/155#issuecomment-568814187

**Reply by Posvar** [reply]
Thanks - this solved it!
